composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "faktiva/prestashop-seo-tk",
+    "name": "faktiva/faktiva_seo_tk",
```

Comment:
I gonna close this PR because the other branch is almost implemented 
https://github.com/faktiva/prestashop-seo-tk/tree/fix-ps-composer-support